api-client/package.json

```
@@ -32,6 +32,7 @@
     "@types/normalize-path": "^3.0.0",
     "@types/react": "16.14.0",
     "@types/seedrandom": "^3.0.0",
+"@types/geojson": "^7946.0.8",
```

```suggestion
   "@types/geojson": "^7946.0.8",
```

or just use auto format :)